tools/docker/build-env-tf.sh

```diff
@@ -0,0 +1,42 @@
+# TF - bazel build env variables
+
+export GCC_HOST_COMPILER_PATH=$(which gcc)
+# Optimization is controlled with bazel --config=opt, it will set AVX and SSE flags on linux if possible (without the march=native flag)
+# It is disabled in TF2.4 config, you may pass it with --copt='-march=native', see BZL_CONFIG arg or just uncomment this line
+#export CC_OPT_FLAGS="-march=native -Wno-sign-compare"
+export PYTHON_BIN_PATH=$(which python)
+export PYTHON_LIB_PATH="$($PYTHON_BIN_PATH -c 'import site; print(site.getsitepackages()[0])')"
+export TF_ENABLE_XLA=1
+export TF_NEED_MPI=0
+export TF_NEED_GDR=0
+export TF_NEED_KAFKA=0
+export TF_NEED_OPENCL=0
+export TF_NEED_JEMALLOC=1
+export TF_NEED_VERBS=0
+export TF_NEED_OPENCL_SYCL=0
+export TF_NEED_COMPUTECPP=0
+export TF_NEED_ROCM=0
+export TF_SET_ANDROID_WORKSPACE=0
+# We need to set BZL_CONFIG=" --config=nogcp --config=noaws --config=nohdfs"
+#export TF_NEED_S3=0
+#export TF_NEED_AWS=0
+#export TF_NEED_GCP=0
+#export TF_NEED_HDFS=0
+# For MKL support BZL_CONFIG+=" --config=mkl --config=opt --copt='-mfpmath=both'"
+#export TF_DOWNLOAD_MKL=1
+#export TF_NEED_MKL=0
+
+# GPU
+export TF_NEED_CUDA=0
+export CUDA_TOOLKIT_PATH=$(find /usr/local -maxdepth 1 -type d -name 'cuda-*')
+if  [ ! -z $CUDA_TOOLKIT_PATH ] ; then
+    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$CUDA_TOOLKIT_PATH/lib64:$CUDA_TOOLKIT_PATH/lib64/stubs"
```

Comment:
This makes me think that you have kept the fix provided in #24 (see @Pratyush1991 comment [here](https://github.com/remicres/otbtf/issues/24#issuecomment-658538475)) so it's great new!